concordium-contracts-common/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased changes
 
+- Implement `quickcheck::Arbitrary` for `Timestamp`, `AccountAddress`, `ContractAddress`, `Address`,  `ChainMetadata`, `AttributeTag`, `AttributeValue` and `OwnedPolicy`.
```

Comment:
Please merge **main** into this branch and resolve the conflict here. The last contracts-common release is 5.0.0

Reply:
Done